qmf/src/plugins/messageservices/imap/imapclient.cpp

```diff
@@ -897,9 +898,17 @@ void ImapClient::commandTransition(ImapCommand command, OperationStatus status)
                 && _pushConnectionsReserved) {
                 _waitingForIdle = true;
                 emit updateStatus( tr("Logging in idle connection" ) );
+#ifdef USE_ACCOUNTS_QT
+                if (_ssoSessionManager) {
+                    ssoProcessLogin();
+                } else {
```

Comment:
The approach in this code is that if manager is not available(it should be constructed when acct is set), we default to normal password from accounts, not the clearest but is like that in the other places already.